Userland/Libraries/LibWeb/CSS/CSSPropertyRule.cpp

```diff
@@ -36,19 +36,39 @@ void CSSPropertyRule::initialize(JS::Realm& realm)
 // https://www.w3.org/TR/cssom-1/#serialize-a-css-rule
 String CSSPropertyRule::serialized() const
 {
-    // Note: The spec doesn't cover this yet. Using the WPT `cssText` tests instead for inspiration
-    // Should be pretty close!
-
     StringBuilder builder;
 
-    builder.appendff("@property {}", serialize_an_identifier(name()));
-    builder.append(" {"sv);
-    builder.appendff(" syntax: \"{}\";", syntax());
-    builder.appendff(" inherits: {};", inherits());
+    // Serialization algorithm is defined in the spec below
+    // https://drafts.css-houdini.org/css-properties-values-api/#the-css-property-rule-interface
+
```

Comment:
Please move these changes into the first commit, where `CSSPropertyRule::serialized()` was added.